src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationHandlers.Revocation.cs

```diff
@@ -56,14 +111,26 @@ public ValueTask HandleAsync(PrepareRevocationRequestContext context)
                     throw new ArgumentNullException(nameof(context));
                 }
 
-                // Weibo, VK ID and Yandex don't support the standard "token" parameter and
+                // Webflow, Weibo, VK ID and Yandex don't support the standard "token" parameter and
                 // require using the non-standard "access_token" parameter instead.
-                if (context.Registration.ProviderType is ProviderTypes.Weibo or ProviderTypes.VkId or ProviderTypes.Yandex)
+                if (context.Registration.ProviderType is ProviderTypes.Webflow or ProviderTypes.Weibo or ProviderTypes.VkId or ProviderTypes.Yandex)
                 {
                     context.Request.AccessToken = context.Token;
                     context.Request.Token = null;
                     context.Request.TokenTypeHint = null;
                 }
+                
+                // Miro uses a JSON payload that expects the "accessToken", "clientId", and "clientSecret" properties
+                else if (context.Registration.ProviderType is ProviderTypes.Miro)
+                {
+                    context.Request["accessToken"] = context.Token;
```

Comment:
Since you're going to send the access token in the `Authorization` header, there's no real point mapping it here. And if you don't nullify `context.Request.Token` here, you can reuse the same branch as the `Zendesk` provider 😃

Reply:
Ah yeah, I thought that branch was for the Linear provider... that uses a single `access_token` parameter 🤣